### ISSUE TITLE
Make the site web a11y

### ISSUE THREAD

**Issue by BlaiseBaptist** [BlaiseBaptist/blaisebaptist.github.io#6]
The site does not have very much accessibility and it needs. We need to make the css up to spec and make sure the links are good too.
